model/network.yaml

```
@@ -25,6 +25,10 @@ groups:
           [OSI Transport Layer](https://osi-model.com/transport-layer/) or
           [Inter-process Communication method](https://en.wikipedia.org/wiki/Inter-process_communication).
           The value SHOULD be normalized to lowercase.
+
+          Consider always setting the transport when setting a port number, since
```

Addressing: https://github.com/open-telemetry/semantic-conventions/pull/289/files#r1310991535

```suggestion
          The value SHOULD be normalized to lowercase.
          Consider always setting the transport when setting a port number, since
```

No line breaks allowed in brief, but I also don't want to hide it in a "note"